spec/author_spec.rb

```diff
@@ -1,49 +1,29 @@
 require 'rails_helper'
-
 RSpec.describe Author, type: :model do
-  subject { Author.new(name: 'Ahmad', photo: 'image', bio: 'Teacher from Colombia', posts_counter: 0) }
-
-  before { subject.save }
-
-  context 'name' do
-    it 'name should be Ahmad' do
-      expect(subject.name).to match('Ahmad')
-    end
+  describe 'Author Model' do
+    subject { Author.find(1) }
 
-    it 'name should be present' do
+    it 'checks if subject is invalid if there is no name' do
       subject.name = nil
       expect(subject).to_not be_valid
     end
 
-    it 'name length should be less than 27 characters' do
-      subject.name = 'Ahmad' * 6
-      expect(subject).to_not be_valid
-    end
-  end
-
-  context 'post_counter' do
-    it 'post_counter should be present' do
-      subject.posts_counter = nil
-      expect(subject).to_not be_valid
-    end
-
-    it 'post_counter should be a number' do
-      subject.posts_counter = 'hello'
+    it 'tests post counter to be less than 0' do
+      subject.posts_counter = - 5
       expect(subject).to_not be_valid
     end
   end
 
-  context 'recent_posts' do
-    before(:each) do
-      5.times do |postal|
-        Post.new(title: "Post #{postal}", text: "text #{postal}", comments_counter: 0, likes_counter: 0,
-                 author_id: subject.id)
+  describe 'Author Model methods' do
+    before do
+      6.times do |number|
+        subject.id = 1
+        Post.create(title: 'title', text: "test post #{number}", author: subject)
       end
     end
 
-    it 'should return the 3 latest posts' do
-      expect(subject.three_recent_posts_per_user).to eq(Post.order(created_at: :desc).limit(3))
-      expect(subject.three_recent_posts_per_user).to eq(Post.last(3))
+    it 'loads only the first three posts' do
+      expect(subject.three_recent_posts_per_user.length).to eq(3)
```

Comment:
- [ ] Please fix tests that are failing
![Screenshot from 2022-11-01 19-53-02](https://user-images.githubusercontent.com/44745855/199303954-24f5a796-7be4-4015-b5d7-799ed15a5c23.png)

Reply:
<img width="1440" alt="Screen Shot 2022-11-01 at 10 41 18 PM" src="https://user-images.githubusercontent.com/53530780/199306825-ea169879-e0bf-42b7-a32f-be25d0073f4b.png">

@DeliceLydia 
On my PC, its working and has no problem. what should I do now??

Reply:
@shahira-sadat ooh really? request another review may be another reviewer will not have a problem
and you can share a screenshot